asreview/models/balance/triple.py

```diff
@@ -62,6 +62,7 @@ class TripleBalance(BaseBalance):
     """
 
     name = "triple"
+    label = "Triple"
```

Comment:
Can a bit more information be inlcluded? `This divides the training data into three sets: included papers, excluded    papers found with random sampling and papers found with max sampling`